mex/common/settings.py

```diff
@@ -119,6 +119,16 @@ def get(cls: type[SettingsType]) -> SettingsType:
         description="MEx backend API url.",
         env="MEX_BACKEND_API_URL",
     )
+    backend_api_write_key: SecretStr = Field(
+        SecretStr("dummy_write_key"),
+        description="Backend API key with write access to call POST/PUT endpoints",
+        env="BACKEND_API_WRITE_KEY",
+    )
+    backend_api_read_key: SecretStr = Field(
```

Comment:
why do we need two separate keys for the backend here?

Reply:
We are using only write key for now, but I thought lets implement read key as well for future usage. should I remove read_key from settings then?

Reply:
yeah, then let's go with [yagni](https://de.wikipedia.org/wiki/YAGNI)